packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Configure protocol specific settings
+---
+
+Your app has to support at least one protocol: SAML or OIDC for authentication, or SCIM for provisioning.
+
+If you want to support more than one, you can submit asynchronously. For example, you can submit your SAML app now and then add OIDC configuration details later on when your app is configured for OIDC.
```

Comment:
"add OIDC configuration details later on when your app is configured for OIDC": Might be confusing to talk about updating the app. Is this talking about updating the app integration? I assumed this isn't a case where the app itself will be changed in the future, that the app already supports OIDC.

Reply:
I think the following incorporates the comments here and above:
"Protocol support details can be submitted all together or asynchronously. For example, if your app currently only supports SAML and SCIM, you can create the submission with the SAML and SCIM protocol details. At a later date, when you add OIDC support in your app, you can return to your integration submission, activate the OIDC support panel, and add in the details needed for Okta to enable OIDC support."